metaflow/plugins/cards/ui/src/store.ts

```diff
@@ -5,11 +5,58 @@ import type { Writable } from "svelte/store";
 export const cardData: Writable<types.CardResponse | undefined> =
   writable(undefined);
 
+/**
+ * This function is used to update the cardData object from the window.
+ * It is used to update the cardData object with
+ * changes that have come from the parent window.
```

Comment:
If you're updating comments, probably helpful here to call out that we're intentionally assigning it to the window object as well.